packages/context/src/__tests__/unit/context-config.unit.ts

```diff
@@ -0,0 +1,131 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/context
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {
+  BindingAddress,
+  BindingKey,
+  ConfigurationResolver,
+  Context,
+  DefaultConfigurationResolver,
+  ResolutionOptions,
+  ValueOrPromise,
+} from '../..';
+
+describe('Context binding configuration', () => {
+  /**
+   * Create a subclass of context so that we can access parents and registry
+   * for assertions
+   */
+  class TestContext extends Context {
+    public configResolver: ConfigurationResolver;
+  }
+
+  let ctx: TestContext;
+  beforeEach(createContext);
+
+  describe('configure()', () => {
+    it('configures options for a binding before it is bound', () => {
+      const bindingForConfig = ctx.configure('foo').to({x: 1});
+      expect(bindingForConfig.key).to.equal(
+        BindingKey.buildKeyForConfig('foo'),
+      );
+      expect(bindingForConfig.tagMap).to.eql({config: 'foo'});
+    });
+
+    it('configures options for a binding after it is bound', () => {
+      ctx.bind('foo').to('bar');
+      const bindingForConfig = ctx.configure('foo').to({x: 1});
+      expect(bindingForConfig.key).to.equal(
+        BindingKey.buildKeyForConfig('foo'),
+      );
+      expect(bindingForConfig.tagMap).to.eql({config: 'foo'});
+    });
+  });
+
+  describe('getConfig()', () => {
+    it('gets config for a binding', async () => {
+      ctx.configure('foo').toDynamicValue(() => Promise.resolve({x: 1}));
+      expect(await ctx.getConfig('foo')).to.eql({x: 1});
+    });
+
+    it('gets local config for a binding', async () => {
```

Comment:
What is "local config"? IIUC, you are testing property access, let's use that term in the test name please.

```suggestion
    it('gets a property of the config object for a binding', async () => {
```